tank/albert-base-v2_tf/albert-base-v2_tf_test.py

```diff
@@ -34,34 +36,16 @@ def configure(self, pytestconfig):
         self.module_tester = AlbertBaseModuleTester(self)
         self.module_tester.benchmark = pytestconfig.getoption("benchmark")
 
-    def test_module_static_cpu(self):
-        dynamic = False
-        device = "cpu"
-        self.module_tester.create_and_check_module(dynamic, device)
-
-    @pytest.mark.skipif(
-        check_device_drivers("gpu"), reason=device_driver_info("gpu")
-    )
-    def test_module_static_gpu(self):
-        dynamic = False
-        device = "gpu"
-        self.module_tester.create_and_check_module(dynamic, device)
-
-    @pytest.mark.skipif(
-        check_device_drivers("vulkan"), reason=device_driver_info("vulkan")
-    )
-    def test_module_static_vulkan(self):
-        dynamic = False
-        device = "vulkan"
-        self.module_tester.create_and_check_module(dynamic, device)
+    param_list = get_valid_test_params()
 
-    @pytest.mark.skipif(
-        check_device_drivers("intel-gpu"),
-        reason=device_driver_info("intel-gpu"),
-    )
-    def test_module_static_intel_gpu(self):
-        dynamic = False
-        device = "intel-gpu"
+    @parameterized.expand(param_list, name_func=shark_test_name_func)
+    def test_module(self, dynamic, device):
+        if device == "gpu":
```

Comment:
you shouldn’t need to check for devices capability since it’s already part of get_valid_params

Reply:
Makes sense, I was following the code in test cases and added this check for capabilities which should be needed anymore.